gen-resourcesdocs/README.md

```diff
@@ -0,0 +1,124 @@
+
+The second chapter first describes the main `Pod` resource. The other resources and definitions documented in the chapter are listed in the `otherDefinitions` field.
+
+## Definition Documentation
+
+For each definition (including resources, which are definitions attached to a Group/Version), the fields are listed, with their type and documentation.
+
+### Documentation of composite types
+
+If the type of a field is a composite type, the Definition of this composite type is documented either inline if this Definition is not documented elsewhere, or the type name is a link pointing to the definition of the composite type.
+
+```
+PodSpec
+
+- affinity (Affinity)   <-- composite type
+
+  If specified, the pod's scheduling constraints
+                                                              --\
+  Affinity is a group of affinity scheduling rules.             |
+                                                                |
+  - affinity.nodeAffinity (NodeAffinity) <-- link to definition |
+                                                                |
+    Describes node affinity scheduling rules for the pod.       |    Inline
+                                                                | definition of
+  - affinity.podAffinity (PodAffinity) <-- link to definition   |    Affinity
+                                                                | composite type
+    Describes pod affinity scheduling rules (e.g. co-locate this|
+    pod in the same node, zone, etc. as some other pod(s)).     |
+                                                                |
+  - affinity.podAntiAffinity (PodAntiAffinity) <-- link to def. |
+                                                                |
+    Describes pod anti-affinity scheduling rules (e.g. avoid    |
+    putting this pod in the same node, zone, etc. as some other |
+    pod(s)).                                                  --/
+```
+
+In this example, the `affinity` field type is `Affinity`, which is a composite type. Because the `Affinity` definition is not listed in any of the `otherDefinitions`, this definition is documented inline.
+
+The `nodeAffinity` field type of the `Affinity` composite type is `NodeAffinity`, which is listed in a `otherDefinitions`. For this reason, this composite type is not documented inline, but the `NodeAffinity` is a link to the definition elsewhere in the documentation. Same for the `podAffinity` and `PodAntiAffinity` fields.
+
+### Ordering and categorization of fields
+
+By default, the fields of a Definition are listed in alphabetic order. The `content/<version>/fields.yaml` file is used to order the fields in another order, and also to group fields into categories.
```

Comment:
nit: The fields are rendered in alphabetical order (not listed)?

Reply:
fixed, thanks